packages/ts-migrate-server/src/migrate/index.ts

```diff
@@ -10,21 +10,41 @@ interface MigrateParams {
   rootDir: string;
   tsConfigDir?: string;
   config: MigrateConfig;
+  sources?: string | string[];
 }
 
 export default async function migrate({
   rootDir,
   tsConfigDir = rootDir,
   config,
+  sources,
 }: MigrateParams): Promise<number> {
   let exitCode = 0;
 
   log.info(`TypeScript version: ${ts.version}`);
 
   const serverInitTimer = new PerfTimer();
 
+  if (sources !== undefined) {
+    if (Array.isArray(sources)) {
```

Comment:
nit: instead of multiple level `if` conditions, might be more readable to use a ternary here: 
```sources = Array.isArray(sources) ? sources.map((source) => path.join(rootDir, source)) : [sources]```